src/features/components/molecules/image-text-boxes/index.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react'
+import { TAcuityFeatures } from './types'
+import FlexBox from 'features/components/atoms/flex-box'
+import Typography from 'features/components/atoms/typography'
+import { Localize } from 'components/localization'
+import useRegion from 'components/hooks/use-region'
+import LinkButton from 'features/components/atoms/link-button'
+
+interface TImageWithParagraph {
+    contentData: TAcuityFeatures
+}
+
+const ImageWithParagraph = ({ contentData }: TImageWithParagraph) => {
+    const { is_eu } = useRegion()
+    const { mobileFlexDirection, data } = contentData
+    return (
+        <FlexBox.Box container="fluid" direction={'col'} mt={'20x'} md={{ mt: '40x' }}>
+            {data.map((feature) => {
+                return (
+                    <FlexBox.Box
+                        key={feature?.id}
+                        direction={mobileFlexDirection}
+                        mb={'10x'}
+                        justify={'center'}
+                        align="center"
+                        md={{
+                            direction: feature.direction,
```

Comment:
Instead of passing directions, can we use something like that?
```suggestion
                            direction: (index == 0 || index % 2 == 0) ?  'row' : 'row-reverse',
```
Because content always change placement as on deriv-ez page where we have 3 or 4 parts like on acuity page

Reply:
thats brilliant idea will do this.